action_plugins/cli.py

```diff
@@ -105,6 +105,14 @@ def run(self, tmp=None, task_vars=None):
         socket_path = getattr(self._connection, 'socket_path') or task_vars.get('ansible_socket')
         connection = Connection(socket_path)
 
+        # make command a required argument
+        if not command:
```

Comment:
Same is applicable for `command`. if `command` is not set in the playbook, `None` is getting set by default as given in defaults/main.yaml `network_engine_command: "{{ command | default(None) }}"`.
Hence when not set, it sets the value `''` to the key which is why it is not even hitting the `KeyError` as it gets a key created `command` with a value `''`. We need to have this check.
Otherwise it will return an empty `stdout` with no warning/error.

Reply:
Same as previous